lib/content-services/src/lib/breadcrumb/breadcrumb.component.html

```diff
@@ -57,7 +57,7 @@
         </a>
 
         <div *ngSwitchDefault class="adf-breadcrumb-item-current" aria-current="location">
-            {{ item.name | translate }}
+            {{selectedRowItemsCount < 1 ? (item.name | translate) : ('BREADCRUMB.HEADER.SELECTED' | translate: { count: selectedRowItemsCount }) }}
```

Comment:
I think no need to repeat translate pipe twice. Can be combined into: 
{{ (selectedRowItemsCount < 1 ? item.name : 'BREADCRUMB.HEADER.SELECTED') | translate: { count: selectedRowItemsCount } }}